public/app/store/configureStore.ts

```diff
@@ -35,14 +35,15 @@ export function configureStore() {
     devTools: process.env.NODE_ENV !== 'production',
     preloadedState: {
       navIndex: buildInitialState(),
+      ...(initialState || {}),
     },
```

Comment:
can't this be just: 
```suggestion
    preloadedState: {
      navIndex: buildInitialState(),
      ...initialState,
    },
```

Reply:
Hmm I think you are right for some reason I always think that `...undefined` will fail.